Makefile

```
@@ -0,0 +1,4 @@
+
+.PHONY: localstack
+localstack:
+	docker run --rm -it -e SERVICES="kms" -p 4566:4566 localstack/localstack
```

Can we pin that to the same one used in CI to avoid surprises ?

```suggestion
	docker run --rm -it -e SERVICES="kms" -p 4566:4566 localstack/localstack:4.0.3
```

Another option could be to provide a `docker-compose.yml` file with this service, and just run `docker compose up` here.

So folks using compose in their workflow with integrations in IDEs will be happy (that's obviously my case 😄). That can also allow people to easily override the port with a `docker-compose.override.yml` in case it's in conflict. Kind of unlikely but still.

Totally fine if you don't think it's worth moving to compose, but I think we should pin the version anyway.

---

Another random idea.

Maybe we can also create a `make test` that will run the container with `-d --name=localstack`, then the tests, then tear down the container.

And just call that in CI instead of having two distinct way of running tests.

---

I will just do the pin for now, and in v7 I plan to use Compose in both local and CI.